pkg/apis/litmuschaos/v1alpha1/chaosresult_types.go

```diff
@@ -25,8 +25,12 @@ import (
 // The chaosresult holds the status of a chaos experiment that is listed as an item
 // in the chaos engine to be run against a given app.
 type ChaosResultSpec struct {
-	// Definition carries low-level chaos options
-	ExperimentStatus TestStatus `json:"experimentstatus"`
+	// EngineName defines the name of chaosEngine
+	EngineName string `json:"engine,omitempty"`
+	// ExperimentName defines the name of chaosexperiment
+	ExperimentName string `json:"experiment,omitempty"`
```

Comment:
Can we not use omitEmpty for exoeriment name? It probably is one deterministic and must element for the result - considering both the SoT & EoT